webapp/updates.py

```diff
@@ -347,12 +347,17 @@ def process_list(self, data):
             'update_list': {},
         }
 
+        # Get list of valid repository IDs based on input paramaters
+        available_repo_ids = self._process_repositories(data, response)
+
+        repo_ids_key = hashlib.md5('_'.join([str(id) for id in sorted(available_repo_ids)]).encode('utf-8')).hexdigest()
```

Comment:
_ is a valid character in repo label so something else (space or comma?)  would be better separator.

Reply:
I'm joining repo IDs, so just numbers. Then it goes to the hash function.

Reply:
I see, then it's ok.